source/_components/w800rf32.markdown

```diff
@@ -68,7 +68,7 @@ devices:
       required: false
       type: string
     device_class:
-      description: "The [type or class of the sensor](/components/binary_sensor/) to set the icon in the frontend."
+      description: "The [type or class of the sensor](https://developers.home-assistant.io/docs/en/entity_binary_sensor.html#available-device-classes) to set the icon in the frontend."
```

Comment:
The device class also changes the way the state is referred to. For instance a `door` will be open or closed rather than on or off